.github/workflows/backend-api-push-to-main.yml

```diff
@@ -49,12 +49,13 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
 
-  build-and-push-test-image:
+  build-and-push-test-image-to-dev:
```

Comment:
I think we need to update the `needs` section below to reflect this job name change

Reply:
Thanks!